src/main/java/picard/analysis/directed/RnaSeqMetricsCollector.java

```diff
@@ -95,21 +95,34 @@ public RnaSeqMetricsCollector(final Set<MetricAccumulationLevel> accumulationLev
         return ignoredSequenceIndices;
     }
 
-    private class PerUnitRnaSeqMetricsCollector implements PerUnitMetricCollector<RnaSeqMetrics, Integer, SAMRecord> {
+    protected class PerUnitRnaSeqMetricsCollector implements PerUnitMetricCollector<RnaSeqMetrics, Integer, SAMRecord> {
+
+        protected final RnaSeqMetrics metrics;
```

Comment:
Hi @yfarjoun ,

I'm not sure I understand the question.  The metrics member remains final in this PR.  All I did was make it protected, because otherwise it wasn't accessible in a subclass in a different package.

It isn't feasible (at least not easily) to have an overridable getMetrics(), because the subclass isn't fully initialized until after the base class ctor has returned, and as you can see, the metrics member is accessed in the base class ctor.  I tried an overridable getMetrics() initially, and settled on what I did after discovering that getMetrics() wouldn't work.  Ultimately I think what I've done is cleaner, because there aren't 2 metrics objects for each PerUnitCollector to create confusion.

-Alec